### ISSUE TITLE
Handle accidental submissions 

### ISSUE THREAD

**Issue by arky** [tailwindlabs/flood-map#1]
We need a better way to handle accidental submissions. It is a common problem on touch interfaces: a user might accidentally mark a road as flooded while using zooming, panning touch gestures. 

The current workaround is to tap again to mark the road as flooded. This could be documented in the usage dialog box next to  'Zoom in to your street and click/tap to report it as flooded (or cleared)'.

In the long term, it would be good to provide a change history. The user might choose to revert certain changes before submitting the reports.

**Comment by nickmcintyre** [comment]
@arky Agreed, both on the workaround and the change history. I'm working on a minimal new skin right now that could display instructions on loading. Let's discuss–I'll push in a couple of hours.

**Comment by nickmcintyre** [comment]
@arky have a look at the 'new-skin' branch.

**Comment by arky** [comment]
@nickmcintyre Pulled and tested the 'new-skin' branch on my local machine. The minimal new skin works both on desktop browser and mobile browser modes (tested via devtools). 

It works as expected.

![u-flood-new-skin](https://user-images.githubusercontent.com/808895/30155248-8745f5a2-93e6-11e7-8e32-8c64703164d7.png)

**Comment by arky** [comment]
@nickmcintyre I think it a good idea to group usage instructions into a single chuck. This makes them stand out to the users.  For example: 

> "This is a crowd sourced effort to map inundated roads in _place_. We rely on users to maintain the accuracy of flood reports. Please do your part, especially once roads have dried out.
> 
> **Zoom in to your street and click/tap to report it as flooded or cleared. If you accidentally mark a street, just click/tap again to undo the change.** 
> 
> All data are open.Big thanks to OpenStreetMap India for the code! Learn more."

**Comment by nickmcintyre** [comment]
@arky good call on grouping instructions–just implemented.

**Comment by arky** [comment]
Looks good. :)

At the moment, we can leave this bug (or close it) until we have a change history feature.

**Comment by nickmcintyre** [comment]
@arky I should be able to refactor the submission code (jQuery) and create an 'Upload' button users must hit.
